### ISSUE TITLE
Update the API dictionary

### ISSUE THREAD

**Issue by esloch** [AlertaDengue/AlertaDengue#507]
# Objetivos / Objectivos / Purpose

## Geral / General

- Add the new variables to the API dictionary after inserting them in the **historico_alerta** table:
*[ umidmin | umidmed | tempmed | tempmax ]*
- Depend-on: #506.

**Comment by esloch** [comment]
@claudia-codeco, when it's possible, provide the dictionary with the new data variables.

(@luabida)

**Comment by luabida** [comment]
when collecting data from coppernicus, we would have not only `temp` and `umid`, but also `pressao` and `precip` (atm pressure and rain). Should I keep it in mind when updating the dict?

**Comment by esloch** [comment]
> when collecting data from coppernicus, we would have not only `temp` and `umid`, but also `pressao` and `precip` (atm pressure and rain). Should I keep it in mind when updating the dict?

Yes, we are waiting for the updated dictionary with the new variables.

ping @claudia-codeco.

**Comment by esloch** [comment]
@luabida and @fccoelho, we will continue with this issue open until all variables are included in the dictionary.
